IABTechLab/uid2-self-serve-portal
# Changes to update password page to explain password change

Test:
- give your user in keycloak the update password action in users -> user details -> required user actions
- try to log in and make sure you are redirected to the update password page
- try updating your password to something that doesn't match the requirements and make sure it fails
- update your password
- check that if you login again, you are not redirected to the update password page
